### PR TITLE
Updated Profile variables and Query string variables

### PR DESCRIPTION
**Pull request by Revathi-b**
Updated Profile variables with latest updates and added a new section Query string variables to the document. @AnjanaValsan

### CODE REVIEW

File: MicrosoftSearch/manage-verticals.md [hunk 1]
```diff
@@ -91,12 +91,16 @@ Here are some example queries.
 |Excluding results from archive sites           |NOT (path:http//contoso.sharepoint.com/archive OR path:http//contoso.sharepoint.com/CompanyArchive)|
 | Excluding results based on file type property | NOT(FileType:htm)|  
 
-#### Profile query variables
+### Profile query variables
```
Comment (AnjanaValsan) [inline MicrosoftSearch/manage-verticals.md]: The heading here should be **Query variables**. We are creating a header/section for Query variables and nesting Profile and Query String under it. @Revathi-b

File: MicrosoftSearch/manage-verticals.md [hunk 2]
```diff
@@ -149,7 +153,27 @@ Use the “|” operator to resolve multi-value variables. See the following tab
...
+
+### Query String variables
+You can contextualize search results based on user interaction with SharePoint sites by using Query String variables. Query String variables allows you to pass key-value pairs in the search URL to ensure the search results are customized based on the values passed in the URL.
+
+Let’s understand this feature by exploring a use case. Adam is a Product Manager and uses a SharePoint site to keep his team informed on project updates. As the SharePoint site admin, he wishes to create two button web parts on the site’s home page that allow users to view in-progress tasks and completed tasks. Clicking on the In-Progress web part, deep links users to the ‘Work items’ search vertical, where the results are refined to show only items tagged as **InProgress**.
+
+This can be achieved by specifying the following query in the “Query” section during vertical creation in the administration page.<br />
+`Status:{QueryString.status}` <br /> 
+
+The URL on the SharePoint site widget needs to be updated to pass the following key value pair https://microsoft.sharepoint-df.com/_layouts/15/sharepoint.aspx?v={verticalID}&status=InProgress
```
Comment (AnjanaValsan) [inline MicrosoftSearch/manage-verticals.md:162]: please rename widget to call out its a button web-part

Comment (Revathi-b) [reply MicrosoftSearch/manage-verticals.md:162]: Thanks for pointing out this. I missed it somehow and committing the changes now. Also, web part doesn't have a hyphen @AnjanaValsan

Comment (AnjanaValsan) [reply MicrosoftSearch/manage-verticals.md]: Please update the URL to this. 
https://{your-domain}.[sharepoint.com/sites/](http://sharepoint.com/sites/){site-name}/_layouts/15/search.aspx/{vertical-ID}?status=InProgress

File: MicrosoftSearch/manage-verticals.md [hunk 3]
```diff
@@ -149,7 +153,27 @@ Use the “|” operator to resolve multi-value variables. See the following tab
...
+
+See the following table for more examples of Query String expansion.
```
Comment (AnjanaValsan) [inline MicrosoftSearch/manage-verticals.md]: Please update the table here to use the same format as called out in the document. 
https://microsoft-my.sharepoint-df.com/:w:/p/anvalsan/EcgmfmdnMZtAvJyqKjuo7GsBs_WXxZbISOdSkofIjdKGFg?e=1ptOpk

**Pull request merged**